spec.yml

```diff
@@ -222,25 +265,37 @@ paths:
     post:
       summary: Create a new vault account
       operationId: createVaultAccount
-      description: |
-        Creates a new vault account with the requested name
+      description: >-
```

Comment:
The change from '|' to '>-' after the 'description' keyword causes the UI the render the content in one paragraph without any new lines in between which causes it to look like this:

_Creates a new vault account with the requested name. Learn more in the following [guide](https://developers.fireblocks.com/reference/create-vault-account). Endpoint Permission: Admin, Non-Signing Admin, Signer, Approver, Editor._

Instead of:

_Creates a new vault account with the requested name. Learn more in the following [guide](https://developers.fireblocks.com/reference/create-vault-account). 

Endpoint Permission: Admin, Non-Signing Admin, Signer, Approver, Editor._